src/view/Chart.ts

```diff
@@ -141,18 +141,23 @@ class ChartView {
             plan: renderTaskPlan,
             reset: renderTaskReset
         });
-        this.renderTask.context = {view: this} as SeriesTaskContext;
+        this.renderTask.context = { view: this } as SeriesTaskContext;
     }
 
-    init(ecModel: GlobalModel, api: ExtensionAPI): void {}
+    init(ecModel: GlobalModel, api: ExtensionAPI): void { }
```

Comment:
Please avoid using auto formatting tool on the source code.

Reply:
Sorry, I modified and resubmitted